modules/javafx.graphics/src/main/java/javafx/scene/Node.java

```diff
@@ -10439,37 +10439,24 @@ void releaseAccessible() {
     }
 
     /**
-     * Requests focus traversal from this {@code Node} in the specified direction.
-     * A successful traversal results in the newly focused {@code Node} visibly indicating its focused state.
+     * Tries to move the focus from this {@code Node} in the specified direction.
+     * A successful traversal results in a new {@code Node} being focused.
+     * A focused node is the target of key events and has a visual indicator.
      *
      * @param direction the direction of focus traversal
-     * @return true if traversal was successful
+     * @return {@code true} if traversal was successful
      * @since 24
      */
     public final boolean requestFocusTraversal(TraversalDirection direction) {
-        TraversalDirectionInternal d;
-        switch (direction) {
-        case DOWN:
-            d = TraversalDirectionInternal.DOWN;
-            break;
-        case LEFT:
-            d = TraversalDirectionInternal.LEFT;
-            break;
-        case NEXT:
-            d = TraversalDirectionInternal.NEXT;
-            break;
-        case PREVIOUS:
-            d = TraversalDirectionInternal.PREVIOUS;
-            break;
-        case RIGHT:
-            d = TraversalDirectionInternal.RIGHT;
-            break;
-        case UP:
-            d = TraversalDirectionInternal.UP;
-            break;
-        default:
-            return false;
-        }
-        return TraversalUtils.traverse(this, d, true);
+        TraversalDirectionInternal d = switch(direction) {
+        case DOWN -> TraversalDirectionInternal.DOWN;
+        case LEFT -> TraversalDirectionInternal.LEFT;
+        case NEXT -> TraversalDirectionInternal.NEXT;
+        case PREVIOUS -> TraversalDirectionInternal.PREVIOUS;
+        case RIGHT -> TraversalDirectionInternal.RIGHT;
+        case UP -> TraversalDirectionInternal.UP;
+        default -> null;
```

Comment:
Do we indent labels inside a switch? I find it more readable since `switch` opens a new context. This is also what I remember seeing in OpenJFX. Don't know if it's enforced.

Reply:
> I don't indent case labels inside of switch.

Our typical code style is to indent. I don't have a strong preference, but I switched (pun intended) may years ago from not indenting to always indenting because: A) the IDEs (at least most of them) indent by default; B) most of the code in the JDK and JavaFX does.

> I don't think we enforce one way or the other.

No, we don't. Feel free to leave it as you have it.

Reply:
This is a bit silly.  You have an opening brace, you should be indenting as you would in every other case when an opening brace appears and you break off the line.  So unless there is a **really** good reason to suddenly not do so that has to do with readability, I think this is an really odd stand to take.

Reply:
That isn't an example of "everything on one line". That's effectively the "switch case NNNN:, on a line by itself", which is defensible, although still not preferred.

Anyway, what you currently have is what I called option 1, and it doesn't seem very readable. I'll still leave it up to you, but please consider the comments made in this thread.

Reply:
I'm not sure why this is a counter argument. It's not used in the JDK or JavaFX. I still think option 2 is better, but if we don't enforce a style then it's your choice.

Reply:
I think this discussion is not productive.  Can we focus on essentials please?

Reply:
readability __is__ essential ;) Same as others, wondering why you insist on not going to max it?

Reply:
I agree with readability.  At the same time, every single suggestion is perfectly readable, in my opinion.

The problem is that all this is subjective.  I find the K&R style unreadable, especially when indents are all over the place.  I don't like spaces after if or switch, or inconsistent newlines at the beginning of the class.  And I especially dislike the trailing whitespace rule in jcheck.  But I keep quiet (except this once).

"This is not a productive area of discussion"